draft-ietf-jsonpath-base.md

```diff
@@ -581,7 +581,7 @@ addressed by the root selector `$`.
 #### Examples
 {: unnumbered}
 
-JSON document:
+JSON value:
```

Comment:
I'm wondering if we should introduce examples (which we do show as JSON texts, after all) simply with "JSON: "

Reply:
Make sense. Omit needless words.